docs/VersionMigration.md

```diff
@@ -0,0 +1,36 @@
+# 1.x to 2.0 Migration Guide
+
+## Overview
+
+There are a number of breaking changes from 1.x to 2.0, mostly related to the experience trait system. If you haven't implemented any custom actions or traits, the migration will involve little or no changes in your code.
+
+## General Changes
+
+### Removed
+
+- Properties are no longer supported for anonymous users. `Appcues.anonymous(properties: Map<String, Any>?)` has been superseded by ``Appcues.anonymous()``. If you require properties, use ``Appcues.identify(userId: String, properties: Map<String, Any>? = null)``.
+
+### Changed
+
+- ``AppcuesNavigationDelegate`` is now responsible for handling universal links in addition to scheme links. Consequently, ``AppcuesNavigationDelegate/navigate(to:openExternally:completion:)`` has a new parameter `openExternally` which indicates whether a link is expecting to be opened in an external browser.
```

Comment:
this is the one we chatted about removing for Android side

Reply:
shoot, forgot to remove the line 👎